foreman-release-scl/foreman-release-scl.spec

```diff
@@ -29,6 +29,9 @@ install -m 644 %{SOURCE0} %{buildroot}%{_sysconfdir}/yum.repos.d/
 %config %{_sysconfdir}/yum.repos.d/*.repo
 
 %changelog
+* Mon Apr 09 2018 Eric D. Helms <ericdhelms@gmail.com> 4-2
```

Comment:
Looks like you didn't see my update - I think this is not needed for the 1.17 branch.